packages/grant-explorer/src/features/discovery/RoundDaysDetails.tsx

```diff
@@ -0,0 +1,76 @@
+import tw from "tailwind-styled-components";
+
+export function getRoundDaysText({
+  roundStartsIn,
+  roundEndsIn,
+}: {
+  roundStartsIn?: number;
+  roundEndsIn?: number;
+}) {
+  const days = pluralize(["day", "days"]);
+
+  if (roundStartsIn && roundStartsIn > 0) {
```

Comment:
`undefined | number` used as boolean here.

Reply:
Do you find it improves readability or what's the motivation here to explicitly check for undefined?